conf/machine/imx8mq-evk.conf

```diff
@@ -4,40 +4,47 @@
 #@DESCRIPTION: Machine configuration for NXP i.MX 8M Quad Evaluation Kit
 #@MAINTAINER: Jun Zhu <junzhu@nxp.com>
 
-MACHINEOVERRIDES =. "mx8:mx8m:mx8mq:"
+MACHINEOVERRIDES =. "imx-boot-container:mx8:mx8m:mx8mq:"
 
 require conf/machine/include/imx-base.inc
 require conf/machine/include/arm/armv8a/tune-cortexa53.inc
 
-IMX_DEFAULT_BSP = "nxp"
+IMX_DEFAULT_BSP = "mainline"
```

Comment:
We need to remove this setting here and change imx-base.inc.

Reply:
Make sense, will re-work it in _V2_.